src/loki_alert_rules/high_error_rate.rule

```diff
@@ -0,0 +1,8 @@
+alert: HostHighLogErrorRate
+expr: count_over_time({%%juju_topology%%} |= "error" [1m]) > 0
+for: 1m
+labels:
+  severity: error
+annotations:
+  summary: High error rate in logs (instance {{ $labels.instance }})
```

Comment:
Is this relatively duplicated with the above rules? From the description, it seems to me that they are all trying to detect something based on error logs.

Also, not very sure if the `groups` are allowed here. If it is allowed, I guess it would be better if we could group similar kinds of rules in a `group`.

Reply:
:+1: